apps/heft/UPGRADING.md

```diff
@@ -1,5 +1,25 @@
 # Upgrade notes for @rushstack/heft
 
+### Heft 0.XX.0
```

Comment:
Should I specify a future version here?

Reply:
@iclanton This looks good to me. Do you want to specify the version to place here? This PR is configured for a major version upgrade.

Reply:
We don't want to do a major version just yet, so it would likely be a minor version bump to the next minor version (which should be `0.35.0` I believe)

Reply:
Changed to 0.35.0 now. Also changed the type of my change from `major` to `minor`